### DISCUSSION TITLE
Quote a message from post options menu

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#110] [General Ideas]
User can quote content of some other post to provide more context in conversation. Can be done using pre populating message box with markdown with syntax of quoting

**Comment by agarciamontoro** [comment]
I think @harshilsharma63 has a plugin in progress for that! https://github.com/harshilsharma63/mattermost-plugin-reply/

**Reply by darkLord19** [reply]
Thanks for pointing this out!

**Comment by esethna** [comment]
> User can quote content of some other post to provide more context in conversation. Can be done using pre populating message box with markdown with syntax of quoting

Ya just like GitHub does, love that idea!

**Comment by harshilsharma63** [comment]
It's on the way. I have a webapp PR pending for it though. I need to fix a bunch of tests for this - https://github.com/mattermost/mattermost-webapp/pull/8204

**Reply by esethna** [reply]
Niiiice
